### PR TITLE
 [TEST] Unit test case for NNFW ML-API

### PR DESCRIPTION
**Pull request by gichan-jang**
Unit tests are added to verify NNFW with ML-API.

Detail of the added test case is :
- Single API, Invoke a sample model via Single API
- Invoke a sample model via Pipeline API, with a stream input
- Invoke a sample model via Pipeline API, with two input streams into a single tensor
- Invoke two-sample model via Pipeline API, sharing a single input stream

Signed-off-by: gichan-jang <gichan2.jang@samsung.com>

**Self evaluation:**
1. Build test: [ *]Passed [ ]Failed [ ]Skipped
2. Run test: [ *]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cpp [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * @file        unittest_tizen_capi.cpp
+ * @file        unittest_tizen_nnfw_runtime_raw.cpp
  * @date        13 Mar 2019
  * @brief       Unit test for Tizen CAPI of NNStreamer. Basis of TCT in the future.
```
Comment (myungjoo) [inline tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cpp]: Update date and brief accordingly.

Comment (gichan-jang) [reply tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cpp]: Updated.

Comment (gichan-jang) [reply tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cpp]: I have a question.
Do @date indicates the generated date or revised date?

Comment (dongju-chae) [reply tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cpp]: The first date when you wrote source codes.

**Comment by dongju-chae** [comment]
BTW, why you closed PR #1945? This PR is similar to the previous one.

**Comment by gichan-jang** [comment]
> BTW, why you closed PR #1945? This PR seems to be similar with the previous one.

Yes, It's my mistake. I deleted branch related to #1945...

**Comment by anyj0527** [comment]
Regarding #2004, the file extension should be **unittest_tizen_nnfw_runtime_raw.cc** . Am I right...?

**Comment by gichan-jang** [comment]
> Regarding #2004, the file extension should be **unittest_tizen_nnfw_runtime_raw.cc** . Am I right...?

Updated.

**Comment by dongju-chae** [comment]
> Yes, It's my mistake. I deleted branch related to #1945...

Even if you deleted the branch, you can restore it :)

**Comment by myungjoo** [comment]
Need manual rebase?

File: tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cc [hunk 2]
```diff
@@ -327,7 +1268,12 @@ main (int argc, char **argv)
...
+  /* ignore tizen feature status while running the testcases */
+  set_feature_state (1);
+
   result = RUN_ALL_TESTS ();
 
+  set_feature_state (-1);
+
   return result;
-}
+}
```
Comment (myungjoo) [inline tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cc]: Check your editor configurations.

File: tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cc [hunk 3]
```diff
@@ -1,7 +1,7 @@
 /**
- * @file        unittest_tizen_capi.cc
- * @date        13 Mar 2019
- * @brief       Unit test for Tizen CAPI of NNStreamer. Basis of TCT in the future.
+ * @file        unittest_tizen_nnfw_runtime_raw.cpp
```
Comment (jaeyun-jung) [inline tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cc]: change to .cc

Comment (gichan-jang) [reply tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cc]: Thank you. Updated.

File: tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cc (hunk 2 above)
Comment (gichan-jang) [reply tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cc]: Thank you. Updated.

**Pull request merged**